src/kernel/src/process/module/mod.rs

```diff
@@ -194,7 +194,7 @@ pub(super) struct DebugOpts {
     pub original_mapped_dump: PathBuf,
 }
 
-pub(super) type EntryPoint = extern "sysv64" fn(*mut Arg, extern "sysv64" fn());
+pub(super) type EntryPoint = extern "C" fn(*mut Arg, extern "C" fn());
```

Comment:
Same here. This one is quite complicated because we need to wrap the actual entry point to convert C calling convention to sysv64.